### PR TITLE
Bump Go dependencies

### PR DESCRIPTION
**Pull request by lucacome**
Bumps all go dependencies

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,9 +1,9 @@
 module github.com/rogpeppe/go-internal
 
-go 1.21
+go 1.21.0
```
Comment (ldemailly) [inline go.mod:3]: no no no :) keep it a "language minimal version" and not an "overly specific toolchain version" please
(just my 2 cents/crusade to keep/revert the go line to what it was before 1.21)

**Comment by lucacome** [comment]
@mvdan @rogpeppe can this be merged?

File: go.mod (hunk 1 above)
Comment (ldemailly) [reply go.mod]: resolved? without an answer?

**Review by mvdan — APPROVED** [review]
Thanks - apologies for the slowness.

File: go.mod (hunk 1 above)
Comment (mvdan) [reply go.mod]: This really doesn't matter. Both mean almost exactly the same. If one dependency bumps us from 1.21 to 1.21.0, it's completely fine.

**Pull request merged**